### DISCUSSION TITLE
Console Warning

### DISCUSSION THREAD

**Discussion by SophiaLinetti** [neuefische/questions#812] [Web]
Does anybody knows where i can fix this Warning??

![Bildschirmfoto 2024-02-23 um 11 26 15](https://github.com/neuefische/questions/assets/151823139/318439ac-fdb1-4fd4-8d01-c97f93a7b808)

**Comment by JessicaLoers** [comment]
### More context is needed to answer this question. 
#### _Where and when does the warning occur?_

Your question should always include 
- a Link to the relevant branch, file/line.
- the Explanation of the approaches you've already tried.
- the Error message (please provide it as text, no screenshots!)

👀 see [Capstone Docs](https://web-capstone-docs.neuefische.de/guides/questions)

**Comment by SophiaLinetti** [comment]
its a warning from the browser (FireFox Developer Edtition)

**Comment by josch87** [comment]
- Please provide a link to the relevant branch and the line 216 of touch-simulator.js.

If touch-simulator.js comes with an (NPM) package:
- Check if upgrading to the latest version of the package resolves the issue.
- If upgrading does not resolve the issue, search if there is already a related bug report in the issue tracker system of the package (for example a GitHub issue). If not, report the problem.
- Contribute if it is an open source package/project. Create a new branch from the packages repository and link it to the issue. Fix the warning and create a pull request.
- After the repository owner(s) made a code review, merged your pull request and made a new release including your pull request, upgrade the package in your project.
